modules/tensor_mechanics/doc/content/source/materials/crystal_plasticity/ComputeMultipleCrystalPlasticityStress.md

```diff
@@ -12,50 +12,91 @@ within a continuum mechanics framework enables the utilization of these models
-a locally converged stress, see [#newtonRhapson2ndPK], is computed at each simulation
-iteration on every mesh quadrature point.
+The class `ComputeMultipleCrystalPlasticityStress` is designed to be used in conjunction with
+one or several crystal plasticity models that are derived from the `CrystalPlasticityStressUpdateBase` class.
 
 
-In light of the essential role played by the `CrystalPlasticityUpdate` base class
-in the calculation of the strain and Cauchy stress increment, an overview of the
-algorithm used to calculate and converge the strain and stress increments for a
-general crystal plasticity constitutive model is given below.
+!alert note title=Requires Specific Constitutive Crystal Plasticity Base Class
+Any constitutive crystal plasticity model developed for use with the
+`ComputeMultipleCrystalPlasticityStress` class *must* inherit from the
+`CrystalPlasticityStressUpdateBase` class.
 
-## Algorithm in `CrystalPlasticityUpdate`
+## Crystal Plasticity Governing Equations
 
-Within the `CrystalPlasticityUpdate`  base class, the crystal slip and resulting
+Within the `ComputeMultipleCrystalPlasticityStress`  base class, the crystal slip and resulting
 strain increment are implemented in an updated Lagrangian incremental form. As
 such, all constitutive evolution equations should also be written in the Lagrangian
 form (and related to the reference frame).
 
 The corresponding second Piola-Kirchoff stress measure is used to determine
 local convergence, at each quadrature point, with in the crystal plasticity
 constitutive model base class. Once convergence is achieved within the
-user-specified tolerances, the equivalent Cauchy stress value is calculated and
-passed back to the `ComputeCrystalPlasticityStress` class. The Cauchy stress
-measure is used in the traditional FEM residual calculation within the MOOSE
-framework.
+user-specified tolerances, the equivalent Cauchy stress value is calculated.
+The Cauchy stress measure is then used in the traditional FEM residual calculation within the MOOSE
+framework (see [xtalpl_nr_pk2convergence]).
 
 A brief overview of the relevant continuum mechanics concepts for the crystal
-plasticity constitutive model base class is given before the crystal plasticity
-specific algorithm overview.
+plasticity constitutive model base class is given below.
+
+### Constitutive Equations
+
+Following [!cite](Asaro:1983kf) `ComputeMultipleCrystalPlasticityStress` uses a multiplicative
+decomposition of the deformation gradient into elastic and plastic components:
+\begin{equation}
+  \label{eqn:deformationGradDecomposition}
+  \mathbf{F} = \mathbf{F}^e \mathbf{F}^p
+\end{equation}
+The change in the crystal shape due to dislocation motion is accounted for in
+the plastic deformation gradient tensor, $\mathbf{F}^p$, while the elastic
+deformation gradient tensor, $\mathbf{F}^e$, accounts for recoverable elastic
+stretch and rotations of the crystal lattice. The evolution of the plastic
```

Comment:
Fp isn't a consequence of Lp, it's the other way around. Instead of `\dot{\mathbf{F}}^p = \mathbf{L}^p \mathbf{F}^p`, I would say `\mathbf{L}^p = \dot{\mathbf{F}}^p \mathbf{F}^{p^{-1}}`.

Reply:
This is a good point. I am not sure which way is better to put it here. @sapitts what do you think?

Reply:
I'd argue that it can go either way, we can resolve L^p into P^p, or we can use L^p to find F^p. The latter is what we do in the code, so it is nice to have that represented in the documentation in some way. 
Perhaps a good compromise is to have both the definition of L^P (`\dot{\mathbf{F}}^p = \mathbf{L}^p \mathbf{F}^p`) and then the equation that directly matches what is in the code

Reply:
Sounds good to me. I will make the change accordingly. Thanks @sapitts and @hugary1995